img/r/restorecord.com.svg

```diff
@@ -1 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 720.21 720.21"><g id="a"><rect width="720.21" height="720.21" style="fill:#080808; stroke-width:0px;"/></g><g id="b"><path d="M286.67,220.58l-1.97,13.12c21.65-16.47,44.24-31.22,66.78-44.03,0,0-27.19,9.86-64.82,30.91Z" style="fill:none; stroke-width:0px;"/><path d="M424.02,401.11h-92.74c-5.98,0-9.4,2.99-10.3,8.96l-18.59,124.09c17.56-1.14,30.08-3.94,30.08-3.94,49.74-12.77,91.45-30.15,126.43-49.88l-19.47-44.22-15.41-35.01Z" style="fill:none; stroke-width:0px;"/><path d="M302.39,534.15l18.59-124.09c.89-5.97,4.32-8.96,10.3-8.96h92.74l15.41,35.01,19.47,44.22c20.15-11.37,38.07-23.52,53.97-36.04l-13.67-31.68-7.02-16.25c14.28-3.86,26.71-9.48,37.3-16.86,10.59-7.38,19.59-15.89,27.01-25.56,7.41-9.66,13.36-20.02,17.83-31.09,4.46-11.06,7.54-22.21,9.23-33.46,2.1-14.04,1.93-27.57-.51-40.57-2.45-12.99-7.6-24.32-15.46-33.99-7.86-9.66-18.56-17.39-32.09-23.19-12.98-5.56-29.02-8.38-47.96-8.61-.81-.01-1.55-.09-2.37-.09h-192.85l-5.64,37.62c37.62-21.05,64.82-30.91,64.82-30.91-22.55,12.81-45.14,27.56-66.78,44.03l-1.71,11.43h194.96c17.21,0,29.07,4.66,35.58,13.96,6.51,9.31,8.82,20.29,6.92,32.93-.95,6.32-2.84,12.47-5.66,18.44-2.83,5.98-6.52,11.25-11.06,15.81-4.55,4.57-10.19,8.26-16.94,11.07-6.74,2.82-14.51,4.22-23.29,4.22h-138.58c-19.68,0-34.56,4.4-44.65,13.17-10.1,8.79-16.65,23.18-19.65,43.21l-11.82,78.89-7.64,51.02c20.54,6.9,43.46,7.46,61.23,6.3Z" style="fill:#fff; stroke-width:0px;"/><path d="M123.8,362.41c-83.23,119.62-10.38,202.18,84.69,209.56,0,0-91.88-30.44-68.76-136.32,17.55-80.46,76.5-149.86,144.97-201.94l1.97-13.12c-48.89,27.35-115.41,73.62-162.87,141.83Z" style="fill:#4f46e5; stroke-width:0px;"/><path d="M526.34,475.52c-19.08,15.81-37.32,28.94-52.39,39.02l2.34,5.32,8.26,18.77h69.03l-21.44-49.67-5.8-13.44Z" style="fill:#fff; stroke-width:0px;"/><path d="M633.02,274.51c-4.61,17.23-30.51,99.22-120.15,169.78-15.9,12.52-33.81,24.67-53.97,36.04-34.98,19.73-76.69,37.11-126.43,49.88,0,0-12.52,2.8-30.08,3.94-17.77,1.15-40.69.6-61.23-6.3-27.64-9.29-50.87-30.05-51.03-73.64-.25-67.83,55.75-139.2,125.85-191.95h-35.44c-128.36,109.5-160.44,264.19-52.91,292.65,64.94,17.18,157.2-6.93,237.99-59.28,18.18-11.78,35.76-25.02,52.34-39.51,55.09-48.17,98.82-110.5,115.06-181.59Z" style="fill:#4f46e5; stroke-width:0px;"/><path d="M482.57,170.5c37.56-6.39,93.78-8.85,126.09,23.57,0,0-43.07-84.83-195.46-23.66h67.16c.77,0,1.45.07,2.21.08Z" style="fill:#4f46e5; stroke-width:0px;"/></g></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 720 720"><path fill="#080808" d="M0 0h720v720H0z"/><path fill="none" d="m287 221-2 13c21-17 44-32 66-44 0 0-27 10-64 31ZM424 401h-93c-6 0-9 3-10 9l-19 124c18-1 30-4 30-4 50-13 92-30 127-50l-20-44-15-35Z"/><path fill="#fff" d="m302 534 19-124c1-6 4-9 10-9h93l15 35 20 44c20-11 38-23 54-36l-14-31-7-17a119 119 0 0 0 82-73 121 121 0 0 0 9-74 75 75 0 0 0-48-57c-12-6-29-9-47-9H292l-5 38c37-21 64-31 64-31-22 12-45 27-66 44l-2 11h195c17 0 29 5 36 14 6 9 8 20 6 33 0 6-2 12-5 18s-7 12-11 16c-5 5-10 9-17 11-7 3-15 5-23 5H325c-20 0-35 4-45 13-10 8-16 23-19 43l-12 79-8 51c21 7 44 7 61 6Z"/><path fill="#4f46e5" d="M124 362c-83 120-11 203 84 210 0 0-91-30-68-136 17-81 76-150 145-202l2-13c-49 27-116 73-163 141Z"/><path fill="#fff" d="m526 476-52 39 2 5 9 19h69l-22-50-6-13Z"/><path fill="#4f46e5" d="M633 275c-5 17-30 99-120 169a518 518 0 0 1-180 86c-1 0-13 3-31 4-17 1-40 1-61-6-27-9-51-30-51-74 0-68 56-139 126-192h-35c-129 110-161 264-53 293 65 17 157-7 238-59 18-12 35-25 52-40 55-48 99-110 115-181ZM483 171c37-7 93-9 126 23 0 0-43-85-196-24h70Z"/></svg>
```

Reply:
On the site, it's at a small enough size that it won't be noticeable.
Either way, here's one with a higher numeric precision:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 720 720"><path fill="#080808" d="M0 0h720v720H0z"/><path fill="none" d="m286.7 220.6-2 13.1a602.1 602.1 0 0 1 66.8-44s-27.2 9.8-64.8 30.9ZM424 401.1h-92.7c-6 0-9.4 3-10.3 9l-18.6 124c17.6-1 30-3.9 30-3.9A518.1 518.1 0 0 0 459 480.3l-19.5-44.2-15.4-35Z"/><path fill="#fff" d="m302.4 534.1 18.6-124c.9-6 4.3-9 10.3-9H424l15.4 35 19.5 44.2c20.1-11.3 38-23.5 54-36l-13.7-31.7-7-16.2a118.9 118.9 0 0 0 64.3-42.5c7.4-9.6 13.4-20 17.8-31a148 148 0 0 0 9.3-33.5c2-14 1.9-27.6-.6-40.6a75 75 0 0 0-47.5-57.2c-13-5.5-29-8.3-48-8.6H292.3l-5.6 37.6c37.6-21 64.8-31 64.8-31a600.3 600.3 0 0 0-66.8 44l-1.7 11.5h195c17.2 0 29 4.7 35.5 14a44 44 0 0 1 7 32.9 66 66 0 0 1-5.7 18.4 51.4 51.4 0 0 1-28 26.9 60 60 0 0 1-23.3 4.2H325c-19.6 0-34.5 4.4-44.6 13.2-10.1 8.8-16.7 23.2-19.7 43.2l-11.8 79-7.6 51c20.5 6.8 43.4 7.4 61.2 6.2Z"/><path fill="#4f46e5" d="M123.8 362.4C40.6 482 113.4 564.6 208.5 572c0 0-91.9-30.5-68.8-136.3 17.6-80.5 76.5-150 145-202l2-13.1c-49 27.3-115.4 73.6-162.9 141.8Z"/><path fill="#fff" d="M526.3 475.5c-19 15.8-37.3 29-52.3 39l2.3 5.4 8.3 18.7h69L532 489l-5.8-13.5Z"/><path fill="#4f46e5" d="M633 274.5c-4.6 17.2-30.5 99.2-120.1 169.8a422.7 422.7 0 0 1-54 36 518.1 518.1 0 0 1-126.4 50s-12.6 2.7-30.1 3.9c-17.8 1.1-40.7.6-61.2-6.3-27.7-9.3-51-30.1-51-73.7-.3-67.8 55.7-139.2 125.8-192h-35.5C152.2 371.9 120.1 526.6 227.6 555c65 17.2 157.2-7 238-59.3a459 459 0 0 0 52.4-39.5c55-48.1 98.8-110.5 115-181.6ZM482.6 170.5c37.5-6.4 93.7-8.8 126 23.6 0 0-43-84.9-195.4-23.7h69.4Z"/></svg>
```